pkg/deploy/controller/deploymentconfig/metrics.go

```diff
@@ -0,0 +1,22 @@
+package deploymentconfig
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// DeploymentConfigControllerSubsystem is how this controller is represented in
+// prometheus metrics.
+const DeploymentConfigControllerSubsystem = "deploymentconfig_controller"
+
+var (
+	deploymentConditionCounter = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: DeploymentConfigControllerSubsystem,
+			Name:      "set_condition",
```

Comment:
@smarterclayton it looks ok with the subsystem set properly, probably should be plural (set_conditions)